Dhali-org/Dhali-wallet
# Add XUMM

## What does this PR do

* Adds basic XUMM SignIn functionality

## How should this PR be tested?

* `flutter run`
* Try signing in to XUMM!
* Make sure wallets are mutually exclusive
* Try funding your Dhali balance 

## Checklist before requesting a review
